src/ape/options.py

```diff
@@ -31,3 +35,31 @@ def verbose_option(help=""):
         default=False,
         help=help,
     )
+
+
+class Alias(click.Choice):
+    """Wraps ``click.Choice`` to load account aliases for the active project at runtime."""
+
+    name = "alias"
+
+    def __init__(self, account_type: Type[AccountAPI] = KeyfileAccount):
```

Comment:
Now, different account plugins can import this and initialize it with their own custom type.
This will help make it so you only select the accounts for your type.

A couple PRs from now, I will update the ledger (and trezor) plugins!

Reply:
Can the default be `None` which means "select all"?

Reply:
Ah, that makes sense! I suppose some non-account API related plugins may want to easily allow selection from all accounts.

Reply:
Exactly!